.circleci/config.yml

```diff
@@ -0,0 +1,189 @@
+# PHP CircleCI 2.0 configuration file
```

Comment:
This is pretty much directly copied from https://github.com/Lullabot/mpx-php/blob/master/.circleci/config.yml